pygmt/xarray/backend.py

```diff
@@ -111,9 +145,8 @@ def open_dataset(  # type: ignore[override]
                     vfname=voutfile, kind=raster_kind
                 )
                 # Add "source" encoding
-                source = which(fname=filename_or_obj)
+                source: str | list = which(fname=filename_or_obj, verbose="q")
                 raster.encoding["source"] = (
                     source[0] if isinstance(source, list) else source
```

Comment:
Need to ensure tiles are sorted.
```suggestion
                    sorted(source)[0] if isinstance(source, list) else source
```